src/client/Wrapper.js

```diff
@@ -188,3 +192,7 @@ export default class Wrapper extends React.PureComponent {
     );
   }
 }
+
+export function NoHeaderWrapper(props) {
```

Comment:
Wrapper contains lots of methods and imports that you won't probably use, better just to create a separate component

Reply:
oh nvm just saw @bonustrack comment